<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateTimeWidget</class>
 <widget class="QWidget" name="DateTimeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DateTime and Location</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="horizontalSpacing">
    <number>8</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="dateTimeLabel">
     <property name="text">
      <string>Date&amp;Time:</string>
     </property>
     <property name="buddy">
      <cstring>dateTimeEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="latitudeLabel">
     <property name="text">
      <string>L&amp;atitude:</string>
     </property>
     <property name="buddy">
      <cstring>latitudeDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="latitudeDoubleSpinBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="latitudeLineEdit">
     <property name="inputMask">
      <string>99°99' A; </string>
     </property>
     <property name="text">
      <string>00°00' N</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="longitudeLabel">
     <property name="text">
      <string>L&amp;ongitude:</string>
     </property>
     <property name="buddy">
      <cstring>longitudeDoubleSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="longitudeDoubleSpinBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="longitudeLineEdit">
     <property name="inputMask">
      <string>999°99' A; </string>
     </property>
     <property name="text">
      <string>000°00' E</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="horizontalSpacing">
      <number>8</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0" alignment="Qt::AlignVCenter">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>104</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="autoApplyPushButton">
       <property name="text">
        <string>Auto Apply</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="applyPushButton">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="cycleGroupBox">
     <property name="title">
      <string>cycle</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>8</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="secondsPerCycleDoubleSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accelerated">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string>s per Cycle</string>
        </property>
        <property name="maximum">
         <double>86400.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="cyclePushButton">
        <property name="text">
         <string>&amp;Cycle</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label00">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="timeSlider">
          <property name="maximum">
           <number>1440</number>
          </property>
          <property name="pageStep">
           <number>15</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>60</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label10">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>1.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignHCenter">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>autoApplyPushButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>applyPushButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
